karma.conf.js

```diff
@@ -63,174 +39,66 @@ const SAUCE_LAUNCHERS = {
-      reporter: 'html',
-      ui: 'qunit',
-
-      // Number of milliseconds to wait for an individual `test(...)` function to complete.
-      // The default is 2000.
-      timeout: 10000,
-    },
-  },
+// Files to include in Jasmine tests.
+const FILES_TO_USE = [
+  'packages/*/!(node_modules)/**/!(*.d).ts',
+  'packages/*/!(*.d).ts',
+  'testing/**/*.ts',
+];
 
-  webpackMiddleware: {
-    noInfo: true,
-    stats: 'minimal',
-  },
-};
+// Files to exclude in Jasmine tests.
+const EXCLUDE_FILES = [
+  'packages/**/*.scss.test.ts',
+  'testing/featuretargeting/**',
+  'testing/ts-node.register.js',
+  'scripts/**/*.ts',
+];
 
-const jasmineConfig = {
-  basePath: '',
-  files: FILES_TO_USE,
-  exclude: EXCLUDE_FILES,
-  frameworks: ['jasmine', 'karma-typescript'],
-  karmaTypescriptConfig: {
+module.exports = function(config) {
+  config.set({
+    // Jasmine/TS config.
+    basePath: '',
+    files: FILES_TO_USE,
     exclude: EXCLUDE_FILES,
-    coverageOptions: {
-      threshold: {
-        global: {
-          // TODO: Raise threshold to at least 90% after more tests have been migrated.
-          statements: 80,
-          branches: 70,
-          functions: 50,
-          lines: 80,
-          excludes: [
-            'adapter.ts',
-            'constants.ts',
-            'testing/**/*.ts',
-            'packages/!(mdc-animation)/**/*',
-            'packages/!(mdc-auto-init)/**/*',
-            'packages/!(mdc-base)/**/*',
-            'packages/!(mdc-checkbox)/**/*',
-            'packages/!(mdc-chips)/**/*',
-            'packages/!(mdc-data-table)/**/*',
-            'packages/!(mdc-dialog)/**/*',
-            'packages/!(mdc-dom)/**/*',
-            'packages/!(mdc-drawer)/**/*',
-            'packages/!(mdc-floating-label)/**/*',
-            'packages/!(mdc-form-field)/**/*',
-            'packages/!(mdc-icon-button)/**/*',
-            'packages/!(mdc-line-ripple)/**/*',
-            'packages/!(mdc-linear-progress)/**/*',
-            'packages/!(mdc-list)/**/*',
-            'packages/!(mdc-menu)/**/*',
-            'packages/!(mdc-menu-surface)/**/*',
-            'packages/!(mdc-notched-outline)/**/*',
-            'packages/!(mdc-radio)/**/*',
-            'packages/!(mdc-ripple)/**/*',
-            'packages/!(mdc-select)/**/*',
-            'packages/!(mdc-slider)/**/*',
-            'packages/!(mdc-snackbar)/**/*',
-            'packages/!(mdc-switch)/**/*',
-            'packages/!(mdc-tab-bar)/**/*',
-            'packages/!(mdc-tab-indicator)/**/*',
-            'packages/!(mdc-tab-scroller)/**/*',
-            'packages/!(mdc-tab)/**/*',
-            'packages/!(mdc-textfield)/**/*',
-            'packages/!(mdc-top-app-bar)/**/*',
-          ],
+    frameworks: ['jasmine', 'karma-typescript'],
+    karmaTypescriptConfig: {
+      exclude: EXCLUDE_FILES,
+      coverageOptions: {
+        threshold: {
+          global: {
+            // TODO: Raise threshold to at least 90% after more tests have been migrated.
```

Comment:
is this TODO statement still correct?

Reply:
Depends on the outcome of b/146338340 :) I will work on this next.